rest-api/src/test/java/com/cloudbees/workflow/rest/endpoints/UndefinedWorkflowTest.java

```diff
@@ -65,8 +65,8 @@ public void test_success_flow() throws Exception {
 
     private void assertRunOkay(RunExt workflowRun) {
         Assert.assertEquals(0, workflowRun.getStages().size());
-        Assert.assertEquals(0L, workflowRun.getEndTimeMillis());
+        Assert.assertTrue("Run that should be near instant wasn't", workflowRun.getDurationMillis() < 100);
```

Comment:
Possible, yes.  If that happens, we can increase the timeout (but if it takes >100 millis there's probably something else wrong).